backend-rust/src/graphql_api.rs

```diff
@@ -997,42 +998,56 @@ LIMIT 30", // WHERE slot_time > (LOCALTIMESTAMP - $1::interval)
 }
 
 pub struct Subscription {
-    pub block_added: broadcast::Receiver<Arc<Block>>,
+    pub block_added:      broadcast::Receiver<Block>,
+    pub accounts_updated: broadcast::Receiver<AccountsUpdatedSubscriptionItem>,
 }
+
 impl Subscription {
     pub fn new() -> (Self, SubscriptionContext) {
         let (block_added_sender, block_added) = broadcast::channel(100);
+        let (accounts_updated_sender, accounts_updated) = broadcast::channel(100);
         (
             Subscription {
                 block_added,
+                accounts_updated,
             },
             SubscriptionContext {
                 block_added_sender,
+                accounts_updated_sender,
             },
         )
     }
 }
+
 #[Subscription]
 impl Subscription {
-    async fn block_added(
-        &self,
-    ) -> impl Stream<Item = Result<Arc<Block>, tokio_stream::wrappers::errors::BroadcastStreamRecvError>>
-    {
+    async fn block_added(&self) -> impl Stream<Item = Result<Block, BroadcastStreamRecvError>> {
         tokio_stream::wrappers::BroadcastStream::new(self.block_added.resubscribe())
     }
+
+    async fn accounts_updated(
```

Comment:
The query at the front-end includes an additional `accountAddress` query variable:
```

{
    "id": "1",
    "type": "start",
    "payload": {
        "key": "...",
        "query": "subscription accountsUpdatedSubscription($accountAddress: String!) {\n  accountsUpdated(accountAddress: $accountAddress) {\n    address\n  }\n}",
        "variables": {
            "accountAddress": "4dT5vPrnnpwVrXZgmYLtHrDLvBYhtzheaK4fDWbJewqRCGQKWz"
        },
        "context": {
            "url": "https://api-ccdscan.testnet.concordium.com/graphql",
            "preferGetMethod": false,
            "suspense": false,
            "requestPolicy": "cache-first",
            "meta": {
                "cacheOutcome": "miss"
            }
        }
    }
}
```

Reply:
Right - but what am I supposed to do with that input string? :thinking:

Reply:
The current front-end allows to pass in an `accountAddress` (or the empty string) as a variable:
https://github.com/Concordium/concordium-scan/blob/main/frontend/src/components/Accounts/AccountDetailsContainer.vue#L120

A meaningful logic would be that we either simplify it and not pass a variable here, the notification would be triggered whenever a new row is added to the `affected_accounts` table (option 1).
The front-end uses the following logic anyway:
https://github.com/Concordium/concordium-scan/blob/main/frontend/src/components/Accounts/AccountDetailsContainer.vue#L107

or we keep the variable (require even the variable), the notification would be triggered whenever a new row is added to the `affected_accounts` table and the `affected_account` equals the variable.

Reply:
I chose to implement `option 2` now.